src/pages/Address/styled.tsx

```diff
@@ -82,8 +82,52 @@ export const AddressTransactionsPanel = styled.div`
 `
 
 export const AddressTransactionsPagination = styled.div`
+  display: flex;
+  flex-direction: row;
   margin-top: 4px;
   width: 100%;
+  background-color: white;
+  border-radius: 0 0 6px 6px;
+  box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.12);
+
+  > div {
```

Comment:
I suggest avoiding using tag selectors to specify a specific target whenever possible, as it reduces readability, robustness, and increases cognitive load.
Here, there are clear targets, so corresponding classes or IDs should be added to them.